docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md

```diff
@@ -2,7 +2,11 @@
 title: Pod Security Policies
 ---
 
-_Pod Security Policies_ (or PSPs) are objects that control security-sensitive aspects of pod specification (like root privileges).
+:::caution
+Pod Security Policy is only available in Kubernetes until v1.24. Kubernetes v1.25 and above use [Pod Security Standards](pod-security-standards.md).
+:::
```

Comment:
We may want to highlight that both co-existed in 1.24 (by default), even though it was [Beta](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) back then.

```suggestion
Pod Security Policy is only available in Kubernetes until v1.24. [Pod Security Standards](pod-security-standards.md) is the built-in alternative, available from Kubernetes v1.24.
:::
```